### ISSUE TITLE
Can't use {{VARIABLE}} inside markdown file (inside code blocks)

### ISSUE THREAD

**Issue by pboos** [Requarks/wiki#848]
### Actual behavior
Page is not showing at all (with JavaScript error as below)

```
app.5862db58c59c765b6a36.js:117 ReferenceError: MONITORING_HOST is not defined
    at Sn.eval (eval at na (app.5862db58c59c765b6a36.js:117), <anonymous>:3:9475)
    at Sn.e._render (app.5862db58c59c765b6a36.js:117)
    at Sn.r (app.5862db58c59c765b6a36.js:117)
    at pn.get (app.5862db58c59c765b6a36.js:117)
    at new pn (app.5862db58c59c765b6a36.js:117)
    at app.5862db58c59c765b6a36.js:117
    at Sn.$mount (app.5862db58c59c765b6a36.js:117)
    at Sn.$mount (app.5862db58c59c765b6a36.js:117)
    at Sn.e._init (app.5862db58c59c765b6a36.js:117)
    at new Sn (app.5862db58c59c765b6a36.js:117)
```

### Expected behavior
Page should show correctly

### Steps to reproduce the behavior
1. New Page
2. /test
3. Choose Markdown editor
4. Press "ok" at the right top
5. As content use
    ``````
   ```
    {{HELLO}}
    ```
6. Press CREATE
7. ---> NOTHING SHOWING (except javascript error)

### Comments
I would argue, that `{{VARIABLE}}` is something one would often use in code scripts. So it should be allowed in a markdown file. I know you probably want to allow some variables as well. but probably should not? Not sure. This broke several files we imported.

**Comment by NGPixel** [comment]
Fixed by 585f2d029626f4cb87a5c4b94f1bd5e0b70775c7

**Comment by pboos** [comment]
Awesome! Thanks!
